src/templates/service.yaml

```diff
@@ -1,10 +1,20 @@
 apiVersion: v1
 kind: Service
 metadata:
+  labels:
+    app: seldon
+    app.kubernetes.io/instance: 'seldon-core'
+    app.kubernetes.io/name: 'seldon-core-operator'
```

Comment:
We should either drop this annotation, or set it to `{{ self.model.app.name }}`

Reply:
Agree, will use the latter.

Reply:
This is done.